From c03d4ad7ce5a5b2796ad52038706ef74bdbfb512 Mon Sep 17 00:00:00 2001
From: Hivan Du <doo@hivan.me>
Date: Wed, 14 Jan 2015 12:50:12 +0800
Subject: [PATCH] learn format-patch

---
 Git Note.md | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Git Note.md b/Git Note.md
index 13d6f07..fe16b19 100644
--- a/Git Note.md	
+++ b/Git Note.md	
@@ -503,10 +503,11 @@ Git项目本身提供了一份文档(Git项目源代码目录中Documentation/Su
 	cd project
 	git checkout -b featureA
 ### 公开的大型项目
-
+可以用`git farmat-patch`来生成`mbox`格式的文件然后作为附件发送。
+每个提交都会封装为一个`.patch`后缀的`mbox`文件，
 ## 项目的管理
 
-
+### 使用特性分支进行工作
 
 
 
-- 
2.1.3

